license-tool.toml

```diff
@@ -0,0 +1,2 @@
+[overrides]
```

Comment:
I don't recall what does this do. Did you run `dd-rust-license-tool`?

Reply:
dd-rust-licence-tool fails without this config because "crunchy" package doesn't have repository config in cargo.toml
https://github.com/eira-fransham/crunchy/issues/9